src/memo/memo.ml

```diff
@@ -747,7 +741,7 @@ end
    Below are some notes on how/why this algorithm works.
 
    By "computation" we mean execution of a "shared fiber", which we represent by
-   a [Once.t]. Such computations can be in one of three states: not started,
+   a [Computation.t]. Computations can be in one of three states: not started,
```

Comment:
This comment should probably be updated: computations can no longer be "not started", I assume?

Reply:
Oh, never mind, we still have `Computation.create` and `Computation.force` as separate functions. 
I guess that's a bit tricky to eliminate.

Reply:
Yeah, I tried eliminating it but the code gets a bit convoluted as a result, so I left `create` separate.